installer/conf/container.conf

```diff
@@ -23,6 +23,14 @@
   log_level debug
 </source>
 
+#custom_metrics_mdm filter plugin
+<filter mdm.cadvisorperf**>
+  type filter_cadvisor2mdm
+  custom_metrics_azure_regions eastus,southcentralus,westcentralus,westus2,southeastasia,northeurope,westEurope
+  metrics_to_collect cpuUsageNanoCores,memoryWorkingSetBytes,memoryRssBytes
+  log_level info
```

Comment:
can u change the log level to debug to be consistent ? Are you logging anything @info level to be info ?